src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -284,7 +284,7 @@ export function PriceChart({ width, height, prices }: PriceChartProps) {
           message={prices && prices.length === 0 ? <NoV3DataMessage /> : <MissingDataMessage />}
         />
       ) : (
-        <svg width={width} height={graphHeight}>
+        <svg width={width} height={graphHeight} style={{ minWidth: '100%' }}>
```

Comment:
I think that width governs the internal svg axes (ie for drawn paths), and minWidth governs the scaled DOMElement size. minWidth is necessary in the case that the passed width is 0, to avoid a frame with no axis drawn on it.